aiomcache/client.py

```diff
@@ -236,6 +271,9 @@ async def _storage_command(self, conn: Connection, command: bytes, key: bytes,
         elif exptime < 0:
             raise ValidationException('exptime negative', exptime)
 
+        if flags == 0 and self._set_flag_handler and not isinstance(value, bytes):
+            value, flags = await self._set_flag_handler(value)
```

Comment:
just thinking, probably to be safer flags should be None to check if it was set by user or not, then if not flag handler set to 0 if None

Reply:
Isn't it an integer received from memcached?

I notice that all the methods below have `flags = 0` hardcoded. Not sure if those are supposed to be updated with something..

Reply:
I think as mentioned at the bottom of https://github.com/aio-libs/aiomcache/pull/92/#discussion_r1019456523
This was probably in preparation for added support for flags. So, if we're not using it, I think we should remove it.